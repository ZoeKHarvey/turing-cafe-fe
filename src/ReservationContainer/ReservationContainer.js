import React from 'react';
import ReservationCards from '../ReservationCards/ReservationCards'

const ReservationContainer= ({reservations}) => {
  console.log('res', reservations)
  // const favoriteClass = isFavorite ? 'favorite' : 'card'
  const reservationCards = reservations.map(res => {
    return <ReservationCards
      id={res.id}
      name={res.name}
      date={res.date}
      time={res.time}
      numGuests={res.number}
      key={res.id}
    />
  })
  return (
    <section>
     {reservationCards}
    </section>
  )


}

export default ReservationContainer;


