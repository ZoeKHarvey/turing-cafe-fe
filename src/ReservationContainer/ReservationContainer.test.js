import React from 'react';
import { shallow } from 'enzyme';
import ReservationContainer from './ReservationContainer';

describe('Reservation Container', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    let reservations = [{ name: 'Dog', date: '2019/04/04', time: '4:00', number: 4, id: 14 }, 
    { name: 'Cat', date: '2019/03/03', time: '7:00', number: 2, id: 10 }]
    const wrapper = shallow(<ReservationContainer
      reservations={reservations}
      />);
    expect(wrapper).toMatchSnapshot();
  });
});