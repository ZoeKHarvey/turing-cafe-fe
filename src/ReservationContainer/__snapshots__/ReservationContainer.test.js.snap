// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation Container should match the snapshot with all data passed in correctly 1`] = `
<section>
  <ReservationCards
    date="2019/04/04"
    id={14}
    key="14"
    name="Dog"
    numGuests={4}
    time="4:00"
  />
  <ReservationCards
    date="2019/03/03"
    id={10}
    key="10"
    name="Cat"
    numGuests={2}
    time="7:00"
  />
</section>
`;
